src/crd-controller/HostedServices/V1Alpha2Controller.cs

```diff
@@ -82,7 +85,7 @@ private async Task HandleEvent(WatchEventType @event, KamusSecret kamusSecret)
                         return;
 
                     case WatchEventType.Deleted:
-                        await HandleDelete(kamusSecret);
+                        //await HandleDelete(kamusSecret);
```

Comment:
Why did you comment it out? shouldn't it stay if `setOwnerReference=false` to preserve backward compatible behavior?

Reply:
I thought the purpose of the flag is to make it safer to delete the controller - without side effects. So I think it's better to delete this method.

Reply:
You mean deleting the `KamusSecret` object right?

I think that I agree, it just changes in the behavior that we need to document in the release notes and maybe increase the major version.

Reply:
Lets remove this case and comment though.

Reply:
done